content-src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,23 @@
+const React = require("react");
+
+const Tooltip = React.createClass({
+  render() {
+    const style = {};
+    if (this.props.visible === true) {
```

Comment:
This `visible` property isn't being used or I couldn't find it being set anywhere. That's OK, just observing/curious.